### PR TITLE
Add gofmt error check to CI

### LINKED ISSUE #192: Add gofmt error check to each CI
We need to add the `gofmt` error check on the CI so it will provide prompt failure to the PR authors. It is not reviewer jobs to validate the fmt checks also.

### LINKED ISSUE #197: Duplicate info when executing `crc start` from a stopped state
In the output of `crc start -b <bundle>` from a stopped state (VM exists, but is stopped), the line `INFO Starting stopped VM ...` appears twice. Plus second time offset by a space.

```
crc - Local OpenShift 4.x cluster
INFO Checking if oc binary is cached              
INFO Checking if Virtualization is enabled        
INFO Checking if KVM is enabled                   
INFO Checking if Libvirt is installed             
INFO Checking if user is part of libvirt group    
INFO Checking if Libvirt is enabled               
INFO Checking if Libvirt daemon is running        
INFO Checking if crc-driver-libvirt is installed  
INFO Checking if Libvirt crc network is available 
INFO Checking if Libvirt crc network is active    
INFO Checking if /etc/NetworkManager/conf.d/crc-nm-dnsmasq.conf exists 
INFO Checking if /etc/NetworkManager/dnsmasq.d/crc.conf exists 
INFO Extracting the Bundle tarball ...            
INFO Starting stopped VM ...                      
INFO  Starting stopped VM ...                     
INFO Bridge IP on the host: 192.168.130.1         
INFO Check internal and public dns query ...      
INFO Starting OpenShift cluster ... [waiting 3m]  
INFO To access the cluster as the system:admin user when using 'oc', run 'export KUBECONFIG=/home/jsliacan/.crc/cache/crc_libvirt_4.1.0-rc.5/kubeconfig' 
INFO Access the OpenShift web-console here: https://console-openshift-console.apps-crc.testing 
INFO Login to the console with user: kubeadmin, password: 7cdWj-FtzNP-7VSYE-pZk7H 
INFO Running
```

### PR DESCRIPTION
**Pull request by anjannath**
Fixes #192 
Fixes #197

### CODE REVIEW

File: pkg/crc/machine/machine.go [hunk 1]
```diff
@@ -116,7 +115,7 @@ func Start(startConfig StartConfig) (StartResult, error) {
 		}
 
 		if s != state.Running {
-			logging.InfoF(" Starting stopped VM ...")
+			logging.InfoF("Starting stopped VM ...")
```
Comment (praveenkumar) [inline pkg/crc/machine/machine.go]: This is just fixing the offset but you will still see the message twice.

```
INFO Starting stopped VM ...                      
INFO Starting stopped VM ...                     
```

File: pkg/crc/machine/machine.go [hunk 2]
```diff
@@ -103,7 +103,6 @@ func Start(startConfig StartConfig) (StartResult, error) {
 
 		result.Status = vmState.String()
 	} else {
-		logging.InfoF("Starting stopped VM ...")
```
Comment (anjannath) [inline pkg/crc/machine/machine.go]: This was the other one..

File: pkg/crc/machine/machine.go (hunk 1 above)
Comment (anjannath) [reply pkg/crc/machine/machine.go]: There was another print line which was causing the message to print twice, which is removed https://github.com/code-ready/crc/pull/198/files#r291927668

File: Makefile [hunk 3]
```diff
@@ -91,3 +92,12 @@ clean: clean_docs
...
 integration:
 	go test $(REPOPATH)/test/integration -v --tags=integration --timeout=20m
+
+.PHONY: fmt
+fmt:
+	@gofmt -l -w $(SOURCE_DIRS)
+
+.PHONY: fmtcheck
+fmtcheck: ## Checks for style violation using gofmt
+	@gofmt -l $(SOURCE_DIRS) | grep ".*\.go"; if [ "$$?" = "0" ]; then exit 1; fi
```
Comment (praveenkumar) [inline Makefile]: 👍

File: Makefile [hunk 4]
```diff
@@ -91,3 +92,12 @@ clean: clean_docs
 .PHONY: integration ## Run integration tests
 integration:
 	go test $(REPOPATH)/test/integration -v --tags=integration --timeout=20m
+
+.PHONY: fmt
+fmt:
+	@gofmt -l -w $(SOURCE_DIRS)
```
Comment (praveenkumar) [inline Makefile]: 👍

**Pull request merged**